ui/src/features/annotator/tools/segment-anything-tool/interactive-segmentation-point.component.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { useZoom } from '../../../../components/zoom/zoom';
+import { Circle } from '../../shapes/circle.component';
+import { InteractiveAnnotationPoint } from './segment-anything.interface';
+
+interface InteractiveSegmentationPointProps extends InteractiveAnnotationPoint {
+    isLoading: boolean;
+}
+
+export const InteractiveSegmentationPoint = ({ x, y, positive, isLoading }: InteractiveSegmentationPointProps) => {
+    const zoom = useZoom();
+
+    const fill = positive ? 'var(--brand-moss)' : 'var(--brand-coral-cobalt)';
+    const radius = 1 / zoom.scale;
+
+    return (
+        <>
+            <Circle
```

Comment:
Let's inline the `Circle` component here instead of using the one from `shapes/circle`. I think even for Geti it made a bit more sense to inline it instead of reusing the `Circle` shape (which is associated to an annotation, vs this thing being a UI element for an annotation tool).